### PR TITLE
User manual format+wording changes

### PR DESCRIPTION
**Pull request by LittleVulpix**
Changed formatting on some elements so that it is more consistent
Reworded strange-looking sentences
Fixed some misleading information (please let me know if I'm wrong!)

If you want me to squish these commits or whatever, let me know.

### CODE REVIEW

File: doc/user_manual_en.md [hunk 1]
```diff
@@ -23,16 +23,16 @@ The "Save image" button saves the QR-Code into a image file, while the "Copy ima
...
-+ _Delete_ This button deletes your profile and the corresponding chat history.
-+ _Export_ Allows you to export your profile in a format compatible with other Tox clients. You can also manually backup your *.tox files.
-+ _Logout_ Close your current profile and show the login window.
-+ _Remove password_ If you set a password for your profile, you can remove it with this button.
-+ _Change password_ If you set a password for your profile, you can change the password with this button. You can also set a password for unencrypted profiles here.
++ _Rename:_ Allows you to rename your profile. Your nickname and profile name don't have to be the same.
++ _Delete:_ Deletes your profile and the corresponding chat history.
++ _Export:_ Allows you to export your profile in a format compatible with other Tox clients. You can also manually back up your *.tox files.
++ _Logout:_ Close your current profile and show the login window.
++ _Remove password:_ Removes the existing password for your profile. If profile already has no password, you will be notified.
```
Comment (tux3) [inline doc/user_manual_en.md]: "If profile already has" should be "If **the** profile already has"

Comment (LittleVulpix) [inline doc/user_manual_en.md]: Fixed

**Comment by tux3** [comment]
Looks good to me. Please squash it if you know how, since it's really one logical change.
Thanks!

**Comment by LittleVulpix** [comment]
D: I think that did it. I hope I didn't break it :D

**Comment by LittleVulpix** [comment]
@zetok One more. Sorry about that.

**Comment by LittleVulpix** [comment]
Please don't tell me I have to squish those two extra ones too :D but I'll do it if needs be.

**Comment by zetok** [comment]
Uh. Welp, I've merged it without last fixes..

Well, it would be nice to have `grammar` commit squashed.

**Comment by LittleVulpix** [comment]
I'm really sorry. I spotted those after re-reading the whole damn thing 3 times ... I'm not sure if I can squash the whole thing without breaking your merge, if you've merged already.

**Comment by zetok** [comment]
> I'm really sorry. I spotted those after re-reading the whole damn thing 3 times

No problem, you're not the only one `;)`

> I'm not sure if I can squash the whole thing without breaking your merge, if you've merged already.

You would have to squash only `grammar`, so that only `last minute fixes` would be modified - i.e. `User manual wording+changes` should be left untouched.

That will just result in git history not being as pretty.

**Comment by LittleVulpix** [comment]
Squished. I will not touch this anymore, so please feel free to do what you need to do! And sorry again.

**Pull request merged**